definitions/procedures/restore/configs.rb

```diff
@@ -35,11 +40,17 @@ def restore_configs(backup)
 
       feature(:tar).run(tar_options)
     end
+    # rubocop:enable  Metrics/MethodLength
 
     def reload_configs
       feature(:mongo).reload_db_config if feature(:mongo)
     end
 
+    def online_backup?
```

Comment:
I think, it would be good to handle situation if file itself or `online` key doesn't exist with few logger statements.